components/admin_console/schema_admin_settings.jsx

```diff
@@ -348,6 +348,7 @@ export default class SchemaAdminSettings extends React.Component {
             <RequestButton
                 key={this.props.schema.id + '_text_' + setting.key}
                 requestAction={setting.action}
+                state={this.state}
```

Comment:
This is necessary to get the current state from the action. I couldn't find another way to get the current value of the field, but I'm open to suggestions.

Reply:
Please see my comments above, I don't think this is needed